modules/schematics/src/effect/index.spec.ts

```diff
@@ -274,7 +280,7 @@ describe('Effect Schematic', () => {
     );
     expect(content).toMatch(/export class FooEffects/);
     expect(content).not.toMatch(
-      /loadFoos\$ = this\.actions\$.pipe\(ofType\(FooActionTypes\.LoadFoos\)\);/
+      /loadFoos\$ = this\.actions\$.pipe\(ofType\(FooActionTypes\.LoadFoos/
```

Comment:
Why were these removed?

Reply:
I think we need to rethink this line completely. The purpose is to make sure the root effects class doesn't contain ANY effect at all – no matter whether this is an API effect, the `concatMap(() => EMPTY)` one or the old endless loopy one.
Therefore, should we change this to

```
expect(content).not.toMatch(
  /loadFoos\$ = this\.actions\$.pipe\(/
)
```

to make sure none of the effects is contained?